cfme/tests/webui/test_advanced_search.py

```diff
@@ -108,13 +108,13 @@
         'param', params_values,
         ids=['{}-{}'.format(param.entity, param.destination.lower()) for param in params_values]
     ),
-    pytest.mark.uncollectif(lambda param, appliance: (
-        (param.collection in (MyService, 'physical_providers', 'physical_servers', 'volume_backups',
-                       'volume_snapshots') and appliance.version < '5.9') or
-        (param.collection in (ConfigManager, 'ansible_tower_providers') and
-         appliance.version > '5.10') or
-        (param.filter == 'Template (Ansible Tower) : Name' and appliance.version < '5.10') or
-        (param.filter == 'Job Template (Ansible Tower) : Name' and appliance.version > '5.10')))
+    pytest.mark.uncollectif(
+        lambda param, appliance: (
+            appliance.version > '5.10' and
```

Comment:
looks like `appliance.version` is not needed here

Reply:
Same discussion as above, let's continue discussion there and I'll apply changes to all such statements in this module.